GodotAddinVS/GodotMessaging/MessageHandler.cs

```diff
@@ -44,7 +44,7 @@ protected override async Task<Response> HandleOpenFile(OpenFileRequest request)
 
             var mainWindow = dte.MainWindow;
             mainWindow.Activate();
-            SetForegroundWindow(new IntPtr(mainWindow.HWnd));
+            SetForegroundWindow(mainWindow.HWnd);
```

Comment:
This doesn't compile when targeting VS19. The following should at least build with both versions:

```suggestion
            SetForegroundWindow((IntPtr)mainWindow.HWnd);
```

Whether it works in VS22 at runtime when built from VS19, I don't know. If it doesn't, we could use reflection to do the proper conversion on both versions.

Reply:
It builds, but the main problems is with installing the extension. This problem is changing with `Microsoft.VSSDK.BuildTools` or `Microsoft.VisualStudio.SDK` getting 17.x versions. I would recommend making a 1.1.2 version with VS19 support, which enables either Community, Pro and Enterprise versions to use the extension, and making a 2.0 version with VS22 support. I seperated the two versions locally, which is working right now.